client/my-sites/stats/stats-card-upsell/stats-card-upsell-overlay.tsx

```diff
@@ -17,14 +18,15 @@ const StatsCardUpsell: React.FC< Props > = ( {
 	copyText,
 	buttonLabel,
 	buttonComponent,
+	icon = 'lock',
 } ) => {
 	const translate = useTranslate();
 
 	return (
 		<div className={ clsx( 'stats-card-upsell', className ) }>
 			<div className="stats-card-upsell__content">
 				<div className="stats-card-upsell__lock">
-					<Gridicon icon="lock" />
```

Comment:
We should probably leave an icon as default; otherwise, other usages would break I suppose.

Reply:
Looks like he [added the `lock` icon by default](https://github.com/Automattic/wp-calypso/pull/99278/files/23ccfda9082d79cf18002806d99c41cda565e947#diff-0b7ac73994d3435481c794aadca9dfd750c1b304cef5ce66f6d765fbcd467d29R21).

Reply:
Ah right sorry I missed that!